src/main/java/seedu/address/logic/commands/AddCustomerCommand.java

```diff
@@ -2,73 +2,64 @@
 
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
 import seedu.address.model.person.Customer;
-import seedu.address.model.person.Driver;
 import seedu.address.model.person.Person;
 
 
 
 /**
  * Adds a person to the address book.
  */
-public class AddCustomerDriverCommand extends Command {
+public class AddCustomerCommand extends Command {
 
-    public static final String COMMAND_WORD = "add";
+    public static final String COMMAND_WORD = "addC";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a customer to the address book. "
             + "Parameters: "
-            + PREFIX_CUSTOMER + "CUSTOMER NAME | "
-            + PREFIX_DRIVER + " DRIVER NAME | "
+            + PREFIX_NAME + "CUSTOMER NAME "
             + PREFIX_PHONE + "PHONE "
             + PREFIX_EMAIL + "EMAIL "
             + PREFIX_ADDRESS + "ADDRESS "
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " "
-            + PREFIX_CUSTOMER + "John Doe "
+            + PREFIX_NAME + "John Doe "
             + PREFIX_PHONE + "98765432 "
             + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
             + PREFIX_TAG + "friends "
             + PREFIX_TAG + "owesMoney";
 
-    public static final String MESSAGE_SUCCESS = "New %1$s added: %2$s";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This %1$s already exists.";
+    public static final String MESSAGE_SUCCESS = "New Customer added: %1$s";
+    public static final String MESSAGE_DUPLICATE_PERSON = "This Customer already exists.";
 
     private final Person toAdd;
-    private final String className;
     private final int toAddId;
 
     /**
      * Creates an AddCommand to add the specified {@code Person}
      */
-    public AddCustomerDriverCommand(Person person) {
+    public AddCustomerCommand(Person person) {
         requireNonNull(person);
         toAdd = person;
-        this.className = person.getClass().getSimpleName();
         this.toAddId = person.getId();
     }
 
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
 
-        if (model.hasCustomer(toAddId) || model.hasDriver(toAddId)) {
-            throw new CommandException(String.format(MESSAGE_DUPLICATE_PERSON, className));
+        if (model.hasCustomer(toAddId)) {
+            throw new CommandException(String.format(MESSAGE_DUPLICATE_PERSON));
         }
 
-        if (className.equals(Customer.class.getSimpleName())) {
-            model.addCustomer((Customer) toAdd);
-        } else if (className.equals(Driver.class.getSimpleName())) {
-            model.addDriver((Driver) toAdd);
-        }
-        return new CommandResult(String.format(MESSAGE_SUCCESS, className, toAdd));
+        model.addCustomer((Customer) toAdd);
+        return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd));
```

Comment:
Same here, don't need to typecast, if Customer Class is being used

Reply:
Changed